simulation_server.py

```diff
@@ -324,6 +324,16 @@ def runWebotsInThread(client):
                                     envVarDocker["THEIA_VOLUME"] = volume
                                     envVarDocker["THEIA_PORT"] = port + 500
                                     client.websocket.write_message('ide: enable')
+                                elif info[1].strip() == 'default_controller':
+                                    default_controller = info[2]
+                                    dockerComposePath = config['dockerConfDir'] + "/docker-compose-benchmark.yml"
+                                    envVarDocker["DEFAULT_CONTROLLER"] = default_controller
+                                    envVarDocker["THEIA_PORT"] = port + 500
+                                    client.websocket.write_message('ide: enable')
+                                    os.system(
+                                        f'cp {config["dockerConfDir"]}/remote_controller_launcher.py '
+                                        f'./remote_controller_launcher.py'
```

Comment:
Wouldn't this be simpler?
```suggestion
                                        f'ln -s {config["dockerConfDir"]}/remote_controller_launcher.py'
```

Reply:
Apparently during a Dockerfile build, COPY instructions can't resolve simlinks outside the build context so I have to make a real file copy

Reply:
OK, so let's revert to the copy and add a comment explaining why we cannot make a soft link.

Reply:
Apparently hard links work, I'll use that instead